### PR TITLE
Note on using `--bind`.

### PR DESCRIPTION
**Pull request by pnkfelix**
(I don't actually know if the number of people who might need this warrants having it in this readme. But **I** would have benefited from such a note, so here we are. :smile:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ any web server to check it out in your browser:
 
 The site is now available at <http://0.0.0.0:8000>.
 
+If you need to preview the site from another machine, pass the host's address via `--bind`. For example, if running on 192.168.72.73:
```
Comment (Mark-Simulacrum) [inline README.md:32]: Huh. I would have expected this to work with 0.0.0.0...

Comment (pnkfelix) [reply README.md:32]: Are you saying I should try doing `python3 -m http.server --bind 0.0.0.0` ?

/me tries it.

Okay, you're right, that works too.

(The crucial thing is that it *doesn't* work if you don't pass `--bind` at all)

File: README.md [hunk 2]
```diff
@@ -29,6 +29,15 @@ any web server to check it out in your browser:
...
+
+```console
+> cd site
+> python3 -m http.server --bind 192.168.72.73
```
Comment (pnkfelix) [inline README.md:36]:
```suggestion
> python3 -m http.server --bind 0.0.0.0
```

File: README.md [hunk 3]
```diff
@@ -29,6 +29,15 @@ any web server to check it out in your browser:
...
+```
+
+The site is now available at <http://192.168.72.73:8000>.
```
Comment (pnkfelix) [inline README.md:39]:
```suggestion
The site is now available at <http://192.168.72.73:8000>.

(As a short-cut: If you pass `--bind 0.0.0.0`, the server will be reachable at all of the host's IP addresses.)
```

**Review by Manishearth — APPROVED** [review]

**Pull request merged**